src/js/player.js

```diff
@@ -2481,23 +2481,14 @@ class Player extends Component {
 }
 
 /*
- * Global player list
- *
- * @type {Object}
- */
-Player.players = {};
```

Comment:
This property is probably better object-oriented design than `videojs.getPlayers()` but it creates a circular dependency between video.js and player.js. It also seems like a design flaw to have two locations to access the same information.